experimenter/tests/integration/nimbus/pages/base.py

```diff
@@ -59,3 +80,11 @@ def wait_for_and_find_elements(self, strategy, locator, description=None):
 
     def non_blocking_sleep(self, seconds):
         time.sleep(seconds)
+
```

Comment:
q: isnt non_blocking_sleep actually a blocking sleep?

Reply:
In this case yeah. This was code @jaredlockhart wrote a while back.